tools/sgcspevaluatorcli/tools.go

```diff
@@ -0,0 +1,59 @@
+package sgcspevaluatorcli
+
+import (
+	"context"
+	"os"
+	"os/exec"
+	"path/filepath"
+
+	"go.einride.tech/sage/sg"
+	"go.einride.tech/sage/sgtool"
+)
+
+const name = "csp"
+
+const packageJSONContent = `{
+  "dependencies": {
+    "@einride/csp-evaluator-cli": "^1.0.0"
+  }
+}
```

Comment:
I know this follows the convention of other sage tools, but shouldn't we try to move out these to real `package.json` files so that dependabot watch them for us?

Reply:
good point! i've extracted package.json and pointed dependabot to it